docker/nuxeo-explorer-export-docker/Dockerfile

```diff
@@ -25,7 +25,7 @@ USER root
 COPY --chown=900:0 README.md target/nuxeo-platform-explorer-package-*.zip /packages/
 
 RUN if [ "${USE_LOCAL_EXPLORER}" = "true" ]; then \
-    /install-local-packages.sh /packages >&1; \
+    /install-packages.sh --offline /packages >&1; \
```

Comment:
We are not planning to support folder install anymore  in the `install-packages.sh` (keep it simple), so instead the change should be:

```
COPY --chown=900:0 README.md target/nuxeo-platform-explorer-package-*.zip /packages/nuxeo-platform-explorer-package.zip
...
/install-packages.sh --offline /packages/nuxeo-platform-explorer-package.zip
```

Reply:
That does not simplify things for me where the target directory does not always hold an actual package (hence the presence of the README.md, to avoid a failure on copy).
AFAIK conditional COPY command is not supported --> any other alternative?

Reply:
Use the same `COPY`, but do the rename only if `USE_LOCAL_EXPLORER` is `true`:
```
COPY --chown=900:0 README.md target/nuxeo-platform-explorer-package-*.zip /packages/

RUN if [ "${USE_LOCAL_EXPLORER}" = "true" ]; then \
    mv /packages/nuxeo-platform-explorer-package-*.zip /packages/nuxeo-platform-explorer-package.zip \
    /install-local-packages.sh /packages/nuxeo-platform-explorer-package.zip >&1; \
  elif [ ! -z "${CONNECT_EXPLORER_CLID}" ]; then \
    /install-packages.sh --clid "${CONNECT_EXPLORER_CLID}" --connect-url "${CONNECT_EXPLORER_URL}" ${NUXEO_EXPLORER_PACKAGE} >&1; \
  fi
```